### PR TITLE
Build Hero Demos with Layout

### PR DESCRIPTION
**Pull request by cjwhitedev**
## Jira

[DS-622](https://pegadigitalit.atlassian.net/browse/DS-622)

## Summary

Show Hero examples using Layout

## Details

We want to deprecate and remove the Hero Component, and not have a stand-alone replacement for it. These are demos showing how to use layout in different ways to replace Hero.

## How to test

Pull the branch and compare the new layout hero demos against the Hero Component demos.

## Release notes

Deprecated Hero Component and supplied replacement demos using layout.

### CODE REVIEW

**Review by mikemai2awesome — CHANGES_REQUESTED** [review]
@cjwhitedev Let's convert all usage to twig. As we discussed before, we haven't landed on the naming conventions yet, that's why the Layout docs currently have a warning message. Using twig template will ensure it will continue to work even when we change up the HTML and naming conventions.

**Review by mikemai2awesome — CHANGES_REQUESTED** [review]

File: docs-site/src/pages/pattern-lab/_patterns/50-pages/80-heroes-with-layout/45-centered-content.twig [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set _content %}
```
[the remark this answers is not in the document]
Comment (cjwhitedev) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/80-heroes-with-layout/45-centered-content.twig:1]: They were requested. They details are in the ticket.

[the remark this answers is not in the document]
Comment (mikemai2awesome) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/80-heroes-with-layout/45-centered-content.twig:1]: @cjwhitedev I see. Can you build out exactly what they have in the screenshots? https://pegadigitalit.atlassian.net/browse/WWW-857

[the remark this answers is not in the document]
Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/80-heroes-with-layout/45-centered-content.twig]: @mikemai2awesome I added the live examples to the "Heroes with layout" folder on the doc site. I also moved the existing work to the test folder. Each live site example has a "Notes" section that give integration specifications on how to build each one of the heroes.

**Review by colbytcook — APPROVED** [review]

**Review by mikemai2awesome — APPROVED** [review]

**Pull request merged**